src/test/resources/de/bsi/secvisogram/csaf_cms_backend/integration/fullworkflow.feature

```diff
@@ -0,0 +1,128 @@
+Feature: Hello World
+
+  Background:
+      * url 'http://localhost:8081'
+      * def apiBase = '/api/v1/advisories' 
+
+  Scenario: oauth 2 flow
+    # Login
+	Given url 'http://localhost:9000/realms/csaf/protocol/openid-connect/token'
+	* form field client_id = 'secvisogram'
+	* form field username = 'all'
+	* form field password = 'all'
+	* form field grant_type = 'password'
+	* form field response_type = 'code'
+	* form field audience = 'secvisogram'
+	* form field requested_token_type = 'ID'
+	When method post
+	Then status 200
+	* def accessToken = response.access_token
+	* def refreshToken = response.refresh_token
+	* def session = response.session_state
+
+	########
+	#Upload
+	Given url 'http://localhost:8081'
+	* path apiBase
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	* request read('min.json')
+	When method post
+	Then status 201
+	* def advisoryId = response.id
+	* def revision = response.revision
+
+	########	
+	#Change workflow state to Review
+	#
+	# Draft, Review, Approved, RfPublication, Published
+	* def workflowStatus = 'Review'
+	Given path apiBase + '/' + advisoryId + '/workflowstate/' + workflowStatus
+	* param revision = revision
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	When method patch
+	Then status 200
+	
+	# Get new revision after workflow state change
+	Given path apiBase
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	When method get
+	Then status 200
+	* def filt = function(x){ return x.advisoryId == advisoryId }
+    * def items = get response[*]
+    * def revision = karate.filter(items, filt)[0].revision    
```

Comment:
Shouldn't that be in the same intent?

Reply:
Fixed in https://github.com/secvisogram/csaf-cms-backend/pull/159/commits/2f18434ca7af4783e402c0b5ff70be17bd4e9808